src/config/coti/common.ts

```diff
@@ -0,0 +1,77 @@
+import { AppConfig } from 'config/types';
+import IconCotiLogo from 'assets/logos/cotilogo.svg';
+
+const addresses = {
+  COTI: '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE',
+  ZERO: '0x0000000000000000000000000000000000000000',
+};
+
+export const commonConfig: AppConfig = {
+  mode: 'development',
+  appName: 'Carbon DeFi (Coti)',
+  appUrl: 'https://coti.carbondefi.xyz',
+  carbonApi: 'https://api.carbondefi.xyz/v1/coti/',
+  selectedConnectors: ['MetaMask'],
+  blockedConnectors: ['Tailwind', 'Compass Wallet', 'Seif'],
+  walletConnectProjectId: '',
+  policiesLastUpdated: '31 Jul, 2024',
+  network: {
+    name: 'Coti',
+    logoUrl: IconCotiLogo,
+    chainId: 2632500,
+    blockExplorer: {
+      name: 'CotiScan',
+      url: 'https://mainnet.cotiscan.io',
+    },
+    rpc: {
+      url: 'https://mainnet.coti.io/rpc',
+    },
+    defaultLimitedApproval: true,
+    gasToken: {
+      name: 'Coti',
+      symbol: 'COTI',
+      decimals: 18,
+      address: addresses.COTI,
+      logoURI:
+        'https://assets.coingecko.com/coins/images/2962/standard/Coti.png',
```

Comment:
This logo looks different than the .svg, is it supposed to look the same? It's not circular